server/db/db.js

```diff
@@ -23,7 +23,7 @@ class DBClient {
     this.db = null;
   }
 
-  async runMigration() {
+  async runMigration(useTestDb) {
```

Comment:
I think we should try to avoid knowledge of tests in implementation. Ideally, this kind of thing would be set by the testing framework or test preflight setup and the actual `runMigrations` method is oblivious.

Reply:
If possible, I agree

Reply:
Yeah, makes sense totally. In the beginning, we were not using a makefile to run the tests in the container, and therefore we couldn’t have access to the NODE_ENV.